option optimize_for = LITE_RUNTIME;
import "BlusherData.proto";
message BlusherListAnswer
{
 required uint32 result = 1;
 repeated BlusherData data = 2;
}
